doc/wiki/Installation:-Generate-Makefile.md

```diff
@@ -0,0 +1,521 @@
+# Generate Makefile
+To get the `Makefile`, please execute the following command:
+
+```bash
+python configure.py --machine=your_configuration_file [--your_arguments]
+```
+
+`your_configuration_file` is the configuration file you got from [[Machine Configuration File | Installation:-Machine-Configuration-File]], and `[--your_arguments]` should match your simulation requirements. Please check out [Option List](#option-list) for all the available options.
+
+For example, the following command uses `configs/pleiades.config` machine configuration, sets the FFTW method to `FFTW2`, and enables gravity and GPU.
+
+``` bash
+python configure.py --machine=pleiades --fftw=FFTW2 --gravity=true --gpu=true
+```
+
+> [!TIP]
+> An example script `generate_make.sh` to generate Makefile can be found in each test problem folder,
+e.g., `example/test_problem/Hydro/AcousticWave/generate_make.sh`.
+
+# Option List
+All available options in `configure.py` including compile-time simulation options and `configure.py` only options are listed below.
+* [Configure.py Only](#configurepy-only)
+
+Compile-time simulation options are classified into the following categories:
+* [Physical Modules](#physical-modules)
+* [Hydro](#hydro-options)
+* [ELBDM](#elbdm-options)
+* [Gravity](#gravity-options)
+* [Particles](#particle-options)
+* [Microphysics](#microphysics-options)
+* [In Situ Python Analysis](#in-situ-python-analysis-options)
+* [Parallelization](#parallelization-options)
+* [Miscellaneous](#miscellaneous-options)
+
+> [!CAUTION]
+> Some combinations are mandatory (e.g., `RSOLVER` must be set when
+`--flu_scheme=CTU`), while some combinations are prohibited
+(e.g., `--particle` is not supported when both `--gravity` and `--tracer` are
+disabled). See the "Restriction" of each option carefully.
+
+## `configure.py` Only
+
+| Option | Value | Description |
+|:---:|:---:|---|
+| `-h`        | -               | Show a short help message. |
+| `-lh`       | -               | Show a detailed help message. |
+| `--machine` | Filename string | Select the `*.config` file under `configs` directory. |
+
+## Physical Modules
+
+<table>
+   <tr>
+      <td align=center rowspan=4 width=130px><b>Option</b></td>
+      <td><b>GAMER Name</b></td>
```

Comment:
This is only relevant for developers. So how about moving it to the last row and renaming it as `Corresponding symbolic constant`?